clients/gnmi-ctl/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ add_executable(gnmi-ctl
 
 target_link_libraries(gnmi-ctl
     PUBLIC
-        stratum
+        stratum_hal_lib_tdi
```

Comment:
By linking against the _static_ stratum library, I added only the object files I needed. This adds unneeded files. It may also make us dependent on protobuf files we don't need (assuming that your cli dependencies are correct).

Reply:
Resolved.